src/flow-bin/bin/index.js

```diff
@@ -27,7 +27,15 @@ for (const command of argv) {
   }
 }
 
-const monorepoRoot = findMonorepoRoot();
+const monorepoRoot = (() => {
+  try {
+    return findMonorepoRoot();
+  } catch {
+    // In case it is not used in a monorepo, fallback to process.cwd
+    return process.cwd();
```

Comment:
alternatively, we could accept root as a parameter and use it instead of findMonorepoRoot

Reply:
I think this is fine. But would it be possible to DRY it with `src/flow-bin/src/index.js`? Since it's the same package and the logic should be always the same.

Reply:
🚀